.bazelrc

```diff
@@ -3,6 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # https://docs.opentitan.org/doc/rm/c_cpp_coding_style/#cxx-version specifies
+build --action_env=BAZEL_CXXOPTS="-std=gnu++14"
```

Comment:
interesting that you need this line and the below lines. out of curiosity: is this some problem with rules_rust?

anywho, glad you got this working!

Reply:
Filed https://github.com/bazelbuild/rules_rust/issues/1552